CliClient/tests/StringUtils.js

```diff
@@ -41,4 +41,18 @@ describe('StringUtils', function() {
 		}
 	}));
 
+	it('should find the next whitespace character', asyncTest(async () => {
+		const str = 'Joplin is a free, open source\n note taking and *to-do* application';
+		const testCases = [[0, 6], [12, 17], [23, 29], [48, 54]];
```

Comment:
Likewise, please could you add two test cases:
- for str = ""
- and for when there's no whitespace after the specified index